remoteit/desktop
# [RC-1671] notifications by device

### Description

 notifications by device

### Test plan



1. Sign in
2. Go to devices
3. Select a device and going to device setting pages
4. You should see setting of notification by device

### Ticket

https://remoteit.atlassian.net/browse/RC-1671
